src/inverseRobot.js

```diff
@@ -4,10 +4,22 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotInversed = {};
+
+  for (const key in robot) {
+    const a = robot[key];
+
+    if (robotInversed.hasOwnProperty(a)) {
+      return null;
+    } else {
+      robotInversed[a] = key;
+    }
```

Comment:
if statement returns or passes on, so the code after will always be executed or exited out of function - there is no need to use `else`